apps/api/src/db/dbConnection.ts

```diff
@@ -43,28 +29,7 @@ export const chainDb = new Sequelize(csMap[env.NETWORK], {
   models: chainModels
 });
 
-export const chainDbs: { [key: string]: Sequelize } = Object.keys(chainDefinitions)
-  .filter(x => chainDefinitions[x].connectionString)
-  .reduce(
-    (obj, chain) => ({
-      ...obj,
-      [chain]: new Sequelize(chainDefinitions[chain].connectionString, {
-        dialectModule: pg,
-        logging,
-        logQueryParameters: true,
-        repositoryMode: true,
-        transactionType: DbTransaction.TYPES.IMMEDIATE,
-        define: {
-          timestamps: false,
-          freezeTableName: true
-        },
-        models: getChainModels(chain)
-      })
```

Comment:
We don't need these configs?

Reply:
I wasn't able to find any refs

Reply:
Ok let's just make sure it doesn't break anything :)

Reply:
Well, tests are passing)